### DISCUSSION TITLE
这是前端框架还是全站系统？可以制作api接口吗？

### DISCUSSION THREAD

**Discussion by binwind8** [opentiny/tiny-engine#980] [Q&A]
这个框架包含后端吗？

**Comment by chilingling** [comment]
1. 官网介绍：https://opentiny.design/tiny-engine#/home
2. 公众号、掘金相关介绍：https://juejin.cn/post/7280926568854667299
3. 相关开源后端 repo
	a. https://github.com/opentiny/tiny-engine-backend-java
	b. https://github.com/opentiny/tiny-engine-webservice-java
	c. https://github.com/opentiny/tiny-engine-data-center
	d. https://github.com/opentiny/tiny-engine-webservice

官方微信小助手：opentiny-official，可以添加您到微信交流群交流
